docs/shared_parsers_catalog/neutron_plugin.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Plugin Configuration
+============================
+
+file ``/etc/neutron/plugin.ini``
```

Comment:
OK, following from the docs discussion elsewhere let's get this sorted out before we start trying things out randomly :-) I'll go back to the previous docs system.